public/js/document/printpage.js

```diff
@@ -41,7 +41,9 @@ pimcore.document.printpage = Class.create(pimcore.document.printabstract, {
             this.versions = new pimcore.document.versions(this);
         }
 
-        this.dependencies = new pimcore.element.dependencies(this, "document");
+        if (typeof pimcore.settings.dependency === 'undefined' || pimcore.settings.dependency) {
```

Comment:
by this way, we should be able to avoid enforcing to use admin ui classic to `^1.5` 🤔

Reply:
Should work (hopefully)